azurelinuxagent/common/protocol/restapi.py

```diff
@@ -282,16 +282,16 @@ def get_ext_handlers(self):
     def get_ext_handler_pkgs(self, extension):
         raise NotImplementedError()
 
-    def get_in_vm_artifacts_profile(self):
+    def get_artifacts_profile(self):
```

Comment:
We can always change it, but at the moment this is the only artifact profile. It just seems unnecessary to qualify it as `in_vm_`.